.github/workflows/release.yml

```
@@ -21,22 +21,41 @@ jobs:
         uses: actions/setup-go@v4
         with:
           go-version: '1.23'
+
+      - name: Set up Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.7)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)